test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -3266,6 +3265,7 @@ def forward(self, x):
         self.run_test(MaskedSelectModel(), x)
 
     @skipIfUnsupportedMinOpsetVersion(11)
+    @disableScriptTest()  # dtype not available
```

Comment:
Are these new failures? Is it because of change in shape/type inference?

Reply:
These are new tests. They weren't verified against the new API. Shubham is taking a look. Flagging them here.